2_Operations/work-process.md

```diff
@@ -31,13 +31,15 @@ Primarily we look over the quarterly goals to determine how we are achieving the
 
 The main purpose of the alignment meeting is to adjust our compasses so that we will select the correct user stories in the sprint planning meeting coming the next day. We know we are paddling hard, but are we all paddling in the right direction?
 
-
 ## Commits and Pull Requests
 
 While working on user stories we often need to change code or text that we keep in GitHub. These changes are done with [git commits](https://help.github.com/articles/github-glossary/#commit) and are presented to others for review as [Pull Requests](https://help.github.com/articles/about-pull-requests/).
 
 To ensure that your commits and Pull Requests contain everything necessary for a successful review, read through the [Git](/3_Development/git.md) guide for the best practices.
 
+After issue/user story/pull request is getting close to being completed people should go on zoom and discuss what else needs to be fixed. We don't want to see long PR discussions on GitHub. The **author** of the issue/user story/pull request should ping people on slack and get them on zoom to polish things.
```

Comment:
>After issue/user story/pull request is getting close to being completed people should go on zoom and discuss what else needs to be fixed.

1. Sentence is off. Maybe change `After` to `When an`.

Or `Prior to closing an issue/user story/pull request, people should go on zoom and discuss what else needs to be fixed.`

2. Capitalize `Zoom` and `Slack`.